contrib/cirrus/pr-should-link-jira

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+#
+# Intended for use in CI, for -rhel branches: check git PR, barf if no jira links
+#
+
+ME=$(basename $0)
+
+set -e
+
+# Github label which allows overriding this check
+OVERRIDE_LABEL="No Jira Link"
+
+# Only -rhel branches need jira links
+BRANCH_REGEX="v.*-rhel"
+
+LINK_REGEX="Fixes:? [\\t ]*https:\\/\\/issues.redhat.com\\/[\w\\/-]*"
```

Comment:
Off-hand, I'm not sure bash understands `\w', and I think you probably want `\\w` no?

Reply:
I did some experimenting with $BASH_REMATCH to see what was actually matched, and you are right, its not supported. So I rewrote it to use `[:alnum:]` style matchers instead, which works nicely.